sentry_sdk/integrations/celery.py

```diff
@@ -440,24 +446,27 @@ def sentry_apply_entry(*args, **kwargs):
 
         monitor_config = _get_monitor_config(celery_schedule, app)
 
-        headers = schedule_entry.options.pop("headers", {})
-        headers.update(
-            {
-                "sentry-monitor-slug": monitor_name,
-                "sentry-monitor-config": monitor_config,
-            }
-        )
-
-        check_in_id = capture_checkin(
-            monitor_slug=monitor_name,
-            monitor_config=monitor_config,
-            status=MonitorStatus.IN_PROGRESS,
-        )
-        headers.update({"sentry-monitor-check-in-id": check_in_id})
+        is_supported_schedule = bool(monitor_config)
+        if is_supported_schedule:
```

Comment:
We now only create the "in_progress" checkin for supported schedule types (so if the schedule is sub-minute interval, the check in will not be sent)

Also because the headers are not set in case it is an unsupported type, the ok/failure checkins will also not be sent.